examples/misc_exporter_gltf.html

```diff
@@ -368,7 +368,13 @@
 				// ---------------------------------------------------------------------
 				// Ortho camera
 				// ---------------------------------------------------------------------
-				const cameraOrtho = new THREE.OrthographicCamera( window.innerWidth / - 2, window.innerWidth / 2, window.innerHeight / 2, window.innerHeight / - 2, 0.1, 10 );
+
+				const frustumSize = 2000;
+				const aspect = window.innerWidth / window.innerHeight;
+
+				const cameraOrtho = new THREE.OrthographicCamera( 0.5 * frustumSize * aspect / - 2, 0.5 * frustumSize * aspect / 2, frustumSize / 2, frustumSize / - 2, 0, 2000 );
```

Comment:
That does not look like the correct pattern to me. Something like:
```js
const h = 10; // frustum height
const aspect = window.innerWidth / window.innerHeight;
camera = new THREE.OrthographicCamera( - h * aspect, h * aspect, h, - h, 10, 2000 );
```

Reply:
I'm just using the pattern from `webgl_camera`:

https://github.com/mrdoob/three.js/blob/302240259d4ae8ae54557f7be6f0515a662f256d/examples/webgl_camera.html#L65

TBH, I see no issues with that approach.